src/test/resources/features/refapp-2.x/12-condition/conditions.feature

```diff
@@ -1,43 +1,43 @@
 Feature: Patient's Conditions Managment
 
-  Background:
+  Background: 
     Given User clicks on Conditions from Patient dashboard
     Then System loads Manage Conditions Page
 
-  @selenium
-  @dashboard
+  @selenium @dashboard
   Scenario Outline: Adding an active or inactive condition
     And User clicks on Add new condition
     Then System on Add New Condition Page
     And User enters "<activity>" condition
     And User clicks save
     Then System loads Manage Conditions Page
-    Examples:
+
+    Examples: 
       | activity |
       | active   |
       | inactive |
 
-  @dashboard
+  @selenium @dashboard
   Scenario: Set first condition to inactive
     And User clicks on set inactive button
     Then System should move condition to inactive section
 
-  @dashboard
+  @selenium @dashboard
   Scenario: Set first condition to active
     And User clicks on set active button
     Then System should move condition to active section
 
-  @dashboard
```

Comment:
Why removing this annotation and include them on same line

Reply:
Actually it was just formatting code